### PR TITLE
fix: local peer stats initialisation after migration

### PR DESCRIPTION
**Pull request by eswarclynn**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx [hunk 1]
```diff
@@ -127,12 +128,12 @@ export const StatsForNerds = ({ onOpenChange }) => {
...
 const useTracksWithLabel = () => {
   const tracksMap = useHMSStore(selectTracksMap);
-  const peersMap = useHMSStore(selectPeersMap);
+  const vanillaStore = useHMSVanillaStore();
   const localPeerID = useHMSStore(selectLocalPeerID);
   const tracksWithLabels = useMemo(
     () =>
       Object.values(tracksMap).reduce((res, track) => {
-        const peerName = peersMap[track.peerId]?.name;
+        const peerName = vanillaStore.getState(selectPeerNameByID(track.peerId));
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:136]: does it make a difference compared to peermap?

File: packages/hms-video-store/src/diagnostics/DiagnosticsStatsCollector.ts [hunk 2]
```diff
@@ -35,7 +35,11 @@ export class DiagnosticsStatsCollector {
       localVideoTrackID && this.localVideoTrackStatsList.push(localTrackStats[localVideoTrackID]);
     }
 
-    const subscribeStatsReport = await this.sdk.getWebrtcInternals()?.getSubscribePeerConnection()?.getStats();
+    const subscribeStatsReport = await this.sdk
+      .getWebrtcInternals()
+      ?.getCurrentStats()
```
Comment (raviteja83) [inline packages/hms-video-store/src/diagnostics/DiagnosticsStatsCollector.ts:40]: this chaining is too lengthy too many places/ anyway to reduce?

File: packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx (hunk 1 above)
Comment (eswarclynn) [reply packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx:136]: the fn in useMemo would be recalculated less number of times?

**Review by raviteja83 — APPROVED** [review]

File: packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx (hunk 1 above)
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/StatsForNerds.jsx]: if it is only for name vanilla store js fine, but why does memo have  no deps?

**Review by KaustubhKumar05 — APPROVED** [review]

**Pull request merged**